frontend/src/component/project/Project/ProjectForm/CollaborationModeTooltip.tsx

```diff
@@ -11,25 +13,40 @@ const StyledDescription = styled(Typography)(({ theme }) => ({
     color: theme.palette.text.secondary,
 }));
 
-export const CollaborationModeTooltip: FC = () => (
-    <HelpIcon
-        htmlTooltip
-        tooltip={
-            <>
-                <Box>
-                    <StyledTitle>open: </StyledTitle>
-                    <StyledDescription>
-                        everyone can submit change requests
-                    </StyledDescription>
-                </Box>
-                <Box sx={{ mt: 2 }}>
-                    <StyledTitle>protected: </StyledTitle>
-                    <StyledDescription>
-                        only admins and project members can submit change
-                        requests
-                    </StyledDescription>
-                </Box>
-            </>
-        }
-    />
-);
+export const CollaborationModeTooltip: FC = () => {
+    const privateProjects = useUiFlag('privateProjects');
+    return (
+        <HelpIcon
+            htmlTooltip
+            tooltip={
+                <>
+                    <Box>
+                        <StyledTitle>open: </StyledTitle>
+                        <StyledDescription>
+                            everyone can submit change requests
+                        </StyledDescription>
+                    </Box>
+                    <Box sx={{ mt: 2 }}>
```

Comment:
This could be styled component. You use the same styles also in bottom.

Reply:
Box is already a styled 'div' - SxProps are for overrides - I think it is ok as is - but happy to change it if you think it would be better